Zend/zend_alloc_sizes.h

```diff
@@ -61,4 +60,9 @@
 	_(28, 2560,    8, 5, x, y) \
 	_(29, 3072,    4, 3, x, y)
 
+/* num, size, count, pages */
+#define ZEND_MM_BINS_INFO_0(_, x, y) \
+	_( 0,   16,  256, 1, x, y) \
+	ZEND_MM_BINS_INFO(_, x, y)
```

Comment:
The `8` bytes bin is now a `16` bytes. Removing it entirely complexifies both branches of `zend_mm_small_size_to_bin`.

The separation between ZEND_MM_BINS_INFO and ZEND_MM_BINS_INFO_0 is necessary so that we don't generate duplicate _emalloc_16() / _efree_16() declarations.